tests/unit/test_library_ingress.py

```diff
@@ -60,7 +61,7 @@ class CharmWithConfigDict(NginxIngressConsumerCharm):
             def __init__(self, *args):
                 super().__init__(*args, config_dict=config_dict)
 
-        self.harness = Harness(CharmWithConfigDict, meta=META)
+        self.harness = Harness(CharmWithConfigDict, meta=META)  # type: ignore[arg-type]
```

Comment:
Is it worth describing the problem and why it is reasonable to ignore?

Reply:
Also applies for the other disable rules, see here for the standard: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#static-code-analysis

Reply:
I believe to fix this one would be necessary to change CharmType as covariant = True in the Harness class

https://mypy.readthedocs.io/en/stable/generics.html#variance-of-generic-types
https://ops.readthedocs.io/en/latest/_modules/ops/testing.html#Harness

Reply:
Comment added.